linz-data-importer/help.html

```diff
@@ -1,76 +1,39 @@
-    on GitHub.</p>
-<h3 id="thanks">Thanks</h3>
-<p>Thanks to all those at LINZ who have provided input and feedback.
-    And thanks to Pete King for the icons.</p>
\ No newline at end of file
+<h2 id="get-an-api-key">Get an API key</h2>
+
+<p>The first step in using the LINZ Data Importer is to get an API key. An API key is required from each service you wish to access data from via the plugin.</p>
+
+<h3 id="linz-data-service-and-other-agency-data-portal-api-key">From the LINZ Data Service and other data portals</h3>
+
+<p>See our <a href ="https://www.linz.govt.nz/data/linz-data-service/guides-and-documentation/creating-an-api-key">Creating an API key guide</a> to create an API key for the LINZ Data Service. The same general process can also be used to generate an API key from the agency data portals listed on the About page.</p>
+<p>You must login and create an API key for each individual data source you wish to access data from.</p>
+
+<h3 id="linz-basemaps-api-key">From LINZ Basemaps</h3>
+
+<p>Visit <a href="https://basemaps.linz.govt.nz">LINZ Basemaps</a> to grab an API Key. Simply, open the menu on the right, then copy the pre-populated API key from the full WMTS URL.</p>
+<p>Note: these keys are rotated every 90 days and will need to be updated accordingly. Contact <a href="mailto:basemaps@linz.govt.nz">basemaps@linz.govt.nz</a> if you wish to retrieve an unlimited developer API key.</p>
+
+<h2 id="connecting-to-a-data-source">Connecting to a data source</h2>
+<ol>
+    <li>Go to the Setting tab, click the drop down under the “API Keys” heading and select the domain of the data service which you are interested in (e.g. data.linz.govt.nz).</li>
+    <li>Click the “+” symbol. This will confirm the selected domain and provide a text box to insert the relevant API key.</li>
+    <li>Copy and paste the relevant API key from the service you want to connect to, and click Save.</li>
+    <li>This will trigger the plugin to request the domain’s dataset information, allowing you to view and import the data in QGIS via WMTS and WFS web services.</li>
+    <li>Select the layer of interest and click “Add”. The corresponding dataset will be added to the map display.</li>
+    <li>Another service can then be added by changing the domain and repeating steps 2-5.</li>
+</ol>
+
+<h2 id="technical-information">Technical information</h2>
+
+<h3 id="coordinate-reference-system">Coordinate Reference System (CRS)</h3>
+
+<p>The available CRS options for each layer are shown in a drop down next to the “Add Layer” button (in terms of EPSG code). Select the CRS of the dataset to be requested from the server.</p>
+<p>If the QGIS project initially has a CRS which does not match the imported dataset’s CRS, QGIS will reproject the imported dataset to match the project CRS. <span style="color: #ff0000"><strong>Important</strong></span>: reprojecting the CRS of a dataset can degrade spatial accuracies and relationships.</p>
```

Comment:
There's no need for both `<span>` and `<strong>`, you can simply do `<strong style="color: #ff0000">Important</strong>`.